modules/storages/app/controllers/storages/admin/oauth_clients_controller.rb

```diff
@@ -65,25 +65,12 @@ def create # rubocop:disable Metrics/AbcSize
     service_result.on_failure do
       update_via_turbo_stream(component: Storages::Admin::Forms::OAuthClientFormComponent.new(oauth_client: @oauth_client,
                                                                                               storage: @storage))
```

Comment:
Last(?) problem to be solved: This will not render the `turbo-frame: "page-content"` anymore. So after showing an error, we will not update the next step in the wizard.